### PR TITLE
[zh-cn] Localize blog 2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed

### PR DESCRIPTION
**Pull request by jzhupup**
Added the zh-cn text to this blog:

```
content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md
```

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+<!-- 
+## Scope
+
+### What This Does
+
+Create a periodically auto-refreshing, human and machine-readable list of
+official Kubernetes CVEs 
+-->
+## 作用域
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:56]:
```suggestion
## 范围
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+* Triage and vulnerability disclosure will continue to be done by SRC (Security
+  Response Committee).
+* Listing CVEs that are identified in build time dependencies and container
+  images are out of scope.
+* Only official CVEs announced by the Kubernetes SRC will be published in the
+  feed. 
+-->
+### 不能做什么
+
+* 分类和漏洞披露将继续由 SRC（安全响应委员会）完成。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:74]:
```suggestion
* 脆弱性的分类和披露将继续由 SRC（Security Response Committee，安全响应委员会）完成。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+### Who It's For
+
+* **End Users**: Persons or teams who _use_ Kubernetes to deploy applications
+  they own
+* **Platform Providers**: Persons or teams who _manage_ Kubernetes clusters
+* **Maintainers**: Persons or teams who _create_ and _support_ Kubernetes
+  releases through their work in Kubernetes Community - via various Special
+  Interest Groups and Committees. 
+  -->
+### 谁可以用
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:88]:
```suggestion
### 针对的受众
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+* **最终用户**：**使用** Kubernetes 部署他们的应用程序的个人或团队。
+* **平台提供商**：**管理** Kubernetes 集群的个人或团队。
+* **维护人员**：通过各种特殊兴趣小组和委员会在 Kubernetes 社区中**创建**和**支持** Kubernetes 发布的个人或团队。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:92]:
```suggestion
* **维护人员**：通过各种特殊兴趣小组和委员会在 Kubernetes 社区中**创建**和**支持** Kubernetes
  发布版本的个人或团队。
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+<!-- 
+So in order to improve the feed in future Kubernetes Releases, if you have any
+feedback, please let us know by adding a comment to
+this [tracking issue](https://github.com/kubernetes/sig-security/issues/1) or
+let us know on
+[#sig-security-tooling](https://kubernetes.slack.com/archives/C01CUSVMHPY)
+Kubernetes Slack channel.
+(Join [Kubernetes Slack here](https://slack.k8s.io)) 
+-->
+因此，为了在未来的 Kubernetes 版本中改进订阅源，如果您有任何反馈，请通过添加评论至
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:113]:
```suggestion
因此，为了在未来的 Kubernetes 版本中改进订阅源，如果你有任何反馈，请通过添加评论至
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+[tracking issue](https://github.com/kubernetes/sig-security/issues/1) 告诉我们，
+或者在
+[#sig-security-tooling](https://kubernetes.slack.com/archives/C01CUSVMHPY)
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:116]:
```suggestion
或者在 [#sig-security-tooling](https://kubernetes.slack.com/archives/C01CUSVMHPY)
```

File: content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+Kubernetes Slack 频道上告诉我们，
+(加入[这里的 Kubernetes Slack](https://slack.k8s.io)) 。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md:118]:
```suggestion
（从[这里](https://slack.k8s.io)参与 Kubernetes Slack) 。
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 8]
```diff
@@ -187,22 +189,26 @@ includes the contents of all Secrets.
...
+[ServiceAccount](/docs/reference/access-authn-authz/service-accounts-admin/), granting permission
+to create workloads also implicitly grants the API access levels of any service account in that
+namespace.
 -->
 ### 工作负载的创建 {#workload-creation}
 
-能够创建工作负载的用户（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)） 
-能够访问下层的节点，除非基于 Kubernetes 的
-[Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)做限制。
+在一个命名空间中创建工作负载（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)）的权限隐式地授予了对该命名空间中许多其他资源的访问权限，例如可以挂载在
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:208]:
```suggestion
在一个命名空间中创建工作负载（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)）
的权限隐式地授予了对该命名空间中许多其他资源的访问权限，例如可以挂载在
```

**Review by tengqm — COMMENTED** [review]
please split this into two PRs, each PR should focus on one topic and only one topic.

**Comment by jzhupup** [comment]
@tengqm sorry, I switched the wrong branch, resulting in two file changes, I tried to fix it but didn't fix it.
So I closed this pr and resubmitted  https://github.com/kubernetes/website/pull/36757. Also, all comments have been resolved.

**Pull request closed without merging**